src/main/kotlin/net/lumalyte/lg/interaction/menus/bedrock/BedrockGuildHomeMenu.kt

```
@@ -147,117 +133,35 @@ class BedrockGuildHomeMenu(
-                    // Clean up
-                    activeTeleports.remove(player.uniqueId)
-                    cancel() // Stop the task after successful teleport
-                } else {
-                    // Update action bar with current time remaining
-                    player.sendActionBar(Component.text("§eTeleporting to guild home in §f${currentSession.remainingSeconds}§e seconds..."))
-                    // Decrement after showing the message
-                    currentSession.remainingSeconds--
-                }
+                teleportationService.startTeleport(player, targetLocation)
+                bedrockNavigator.goBack()
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)